shell/src/main/java/org/apache/accumulo/shell/commands/FateCommand.java

```diff
@@ -79,8 +67,8 @@
  * Manage FATE transactions
  */
 public class FateCommand extends Command {
-
-  private final static Logger LOG = LoggerFactory.getLogger(FateCommand.class);
+  private static final String warning =
+      "WARNING: This command is deprecated for removal. Use 'accumulo admin'\n";
```

Comment:
suggest ```
      "WARNING: This command is deprecated for removal. Use 'accumulo admin'  and to use new options\n";
```

Reply:
Was there more to your suggestion? I am only seeing " and to use new options".

Reply:
Suggested the line read `      "WARNING: This command is deprecated for removal. Use 'accumulo admin and to use new options'\n";`

The wording is awkward - but could not think of anything better , but short - this along with the other suggestions, I'm trying to point people to the new options (if the functions are not made available in the current command)

Reply:
I think we accomplished that with the message printing when trying to use the command. I added you as a co-auther on the commit.